packages/plugin/src/theia.d.ts

```diff
@@ -2932,6 +2932,13 @@ export module '@theia/plugin' {
          */
         env?: { [key: string]: string | null };
 
+        /**
+         * A message to write to the terminal on first launch, note that this is not sent to the
+         * process but, rather written directly to the terminal. This supports escape sequences such
+         * a setting text style.
```

Comment:
This message is identical to VSCode's and their version has some copy-editing issues...

```suggestion
         * A message to write to the terminal on first launch. Note that this is not sent to the
         * process, but rather written directly to the terminal. This supports escape sequences such
         * as setting text style.
```